sources/packages/backend/apps/db-migrations/src/migrations/1752517099854-InsertParentDeclarationRequiredParentCanReportNotification.ts

```diff
@@ -0,0 +1,24 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
```

Comment:
Minor. Can we keep the file name(s) shorter to avoid long file names?

e.g. `StudentDeclareParentInformationNotification` and `ParentDeclareParentInformationNotification` as suggestions.